test/vulnz-check/vulnz_check.py

```diff
@@ -0,0 +1,134 @@
+# %%
+from typing import NamedTuple, List
+import os
+
+import kfp
+from kfp import dsl
+from kfp.components import func_to_container_op, InputPath, OutputPath
+
+# %%
+# Vulnerability checking
+
+
+def fetch_image_digest(image: str) -> str:
+    """ Fetch digest of an image.
+
+    Args:
+        image (str): image url
+
+    Returns:
+        str: full image url with sha256 digest
+    """
+    import subprocess
+    import sys
+    digest = subprocess.check_output([
+        'gcloud', 'container', 'images', 'describe', image,
+        "--format=value(image_summary.fully_qualified_digest)"
+    ]).decode(sys.stdout.encoding)
+    return digest
+
+
+fetch_image_digest_component = kfp.components.create_component_from_func(
+    fetch_image_digest, base_image='google/cloud-sdk:alpine'
+)
+
+kritis_check = kfp.components.load_component_from_text(
+    '''
+name: Kritis Check
```

Comment:
In the future we can put this component YAML in /components/. Maybe in `/components/container-operations/Scan_for_vulnerabilities/using_Kritis/`